changelog-system/src/main/java/com/gentics/mesh/changelog/changes/BinaryStorageMigration.java

```diff
@@ -73,11 +73,10 @@ private File dataTargetFolder() {
 	public void applyInTx() {
 		// Move the data directory away
 		try {
-			log.info("Moving current data directory away to {" + dataSourceFolder().getAbsolutePath() + "}");
+			log.info("Moving current data directory away from {" + dataTargetFolder().getAbsolutePath() + "} to {" + dataSourceFolder().getAbsolutePath() + "}");
 			FileUtils.moveDirectory(dataTargetFolder(), dataSourceFolder());
 		} catch (Exception e) {
-			throw new RuntimeException("Could not move data folder to backup location {" + dataTargetFolder().getAbsolutePath()
-				+ "}. Maybe the permissions not allowing this?");
+			throw new RuntimeException("Could not move data folder to backup location {" + dataTargetFolder().getAbsolutePath() + "}.", e);
```

Comment:
Common Logger API does not mix varargs and throwables, so in there cases we have to deal with those ourselves.